xcube_cmems/cmems.py

```diff
@@ -68,16 +71,18 @@ def __init__(self,
         self.opendap_dataset_ids = {}
 
         if not self.cmems_username or not self.cmems_password:
-            raise Exception('CmemsDataStore needs cmems credentials in env vars'
-                            ' CMEMS_USERNAME and CMEMS_PASSWORD or to be '
+            raise Exception('CmemsDataStore needs cmems credentials in env '
+                            'vars CMEMS_USERNAME and CMEMS_PASSWORD or to be '
```

Comment:
```suggestion
            raise ValueError('CmemsDataStore needs cmems credentials in environment '
                            'variables CMEMS_USERNAME and CMEMS_PASSWORD or to be '
```

Always use a specific exception type, never `Exception` directly.
Don't use term "env vars".